povm_toolbox/library/povm_implementation.py

```diff
@@ -182,15 +227,34 @@ def _povm_outcomes(
         povm_metadata: MetadataT,
         loc: int | tuple[int, ...] | None = None,
     ) -> list[tuple[int, ...]]:
-        """TODO."""
+        """Convert the raw bitstrings into POVM outcomes based on the associated metadata.
+
+        Args:
+            bit_array: The raw bitstrings.
+            povm_metadata: The associated metadata.
+            loc: an optional location to splice the bitstrings.
```

Comment:
I am not familiar with the term "splice", was it what you meant ? Or "slice" ?

Reply:
(more occurences below as well)

Reply:
Ah yeh that was a typo. Splicing exists, too, but not in this context. Will change it right away